base_transaction_id/models/account_bank_statement_line.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class AccountBankStatementLine(models.Model):
+
+    _inherit = 'account.bank.statement.line'
+
+    @api.multi
+    def get_reconciliation_proposition(self, excluded_ids=None):
+        """ Look for transaction_ref to give them as proposition move line """
+        if self.name:
```

Comment:
self.ensure_one() ?

Reply:
This was written at a time we considered ensure_one to be called anyway. I can add it thought